warehouse/models/mart/gtfs/fct_daily_scheduled_trips.sql

```diff
@@ -27,17 +32,35 @@ urls_to_gtfs_datasets AS (
     SELECT * FROM {{ ref('int_transit_database__urls_to_gtfs_datasets') }}
 ),
 
+stop_times_grouped AS (
+    SELECT * FROM {{ ref('int_gtfs_schedule__stop_times_grouped') }}
+),
+
 fct_daily_scheduled_trips AS (
     SELECT
         {{ dbt_utils.surrogate_key(['service_index.service_date', 'trips.key']) }} as key,
         service_index.service_date,
         service_index.feed_key,
         service_index.service_id,
+
         trips.key AS trip_key,
+        trips.trip_id AS trip_id,
+
         routes.key AS route_key,
+        routes.route_id AS route_id,
+
         shapes.key AS shape_array_key,
         urls_to_gtfs_datasets.gtfs_dataset_key AS gtfs_dataset_key,
-        trips.warning_duplicate_primary_key AS warning_duplicate_trip_primary_key
+        trips.warning_duplicate_primary_key AS contains_warning_duplicate_trip_primary_key,
+
+        stop_times_grouped.n_stops,
+        stop_times_grouped.n_stop_times,
+        stop_times_grouped.trip_first_departure_ts,
+        stop_times_grouped.trip_last_arrival_ts,
+        stop_times_grouped.service_hours,
+        stop_times_grouped.contains_warning_duplicate_primary_key AS contains_warning_duplicate_stop_times_primary_key,
+        stop_times_grouped.contains_warning_missing_foreign_key_stop_id
+
     FROM int_gtfs_schedule__daily_scheduled_service_index AS service_index
     LEFT JOIN dim_trips AS trips
```

Comment:
This is a bug I created not changed here but this on line 65 should be an `INNER` join instead of left 🤦